### PR TITLE
Add a warning about using same user for cli and web server

### PR DESCRIPTION
**Pull request by pasdeloup**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | --- |

### CODE REVIEW

File: book/installation.rst [hunk 1]
```diff
@@ -228,6 +228,9 @@ If there are any issues, correct them now before moving on.
     configuration (e.g. commonly httpd.conf or apache2.conf for Apache) and setting
     its user to be the same as your CLI user (e.g. for Apache, update the ``User``
     and ``Group`` values).
+    
+    Be careful, this option is not recommended on production environments for security
+    reasons as a compromise server would give to the hacker the same privileges than this user.
```
Comment (snoek09) [inline book/installation.rst]: reasons as a `compromised` server would `give the` hacker the same privileges `as` this user.

**Comment by javiereguiluz** [comment]
I'm not sure about adding this note. My reason is that it implies that this is always  insecure or the "hacker" would get root privileges. But the user shared by the web server and the CLI could/should be very limited and without critical privileges, right?

**Comment by pasdeloup** [comment]
You're right, it's not root privileges, but CLI user usually has more rights than web user. He can launch commands, like app/console doctrine:database:drop even if whatever is done in command can usually be done via web when it's pure php. He can also launch some unsafe binaries in bin/. He's also able to read/write what is generated by the CLI like archives/logs/statistics.
To finish, this user is also usually a real user with a bash, a home directory potentially containing private files.
So I don't like the idea, but maybe a simple warning would be more appropriate than a "never do it"

File: book/installation.rst [hunk 2]
```diff
@@ -228,6 +228,12 @@ If there are any issues, correct them now before moving on.
     configuration (e.g. commonly httpd.conf or apache2.conf for Apache) and setting
     its user to be the same as your CLI user (e.g. for Apache, update the ``User``
     and ``Group`` values).
+    
+    .. caution::
+    
+      If used in a production environment, be sure this user only has limited privileges
+      (no access to private data or servers, launch of unsafe binaries, etc.)
+      as a compromised server would give to the hacker those privileges.
```
Comment (WouterJ) [inline book/installation.rst]: this paragraph needs to be indented with 4 more spaces in order to be part of the caution box.

**Comment by WouterJ** [comment]
:+1: I think a caution might be nice in this place.

File: book/installation.rst (hunk 2 above)
Comment (pasdeloup) [inline book/installation.rst]: It was working, but you're right, and it's better for readability.
I also rebased as my other PR on the same file has just been merged.

**Comment by xabbuh** [comment]
Thank you @pasdeloup.

**Pull request closed without merging**